bin/eva-sub-cli.py

```diff
@@ -34,8 +38,19 @@ def validate_command_line_arguments(args, argparser):
         sys.exit(1)
 
 
+def get_version():
+    base_dir = os.path.abspath(os.path.dirname(main.__file__))
+    version = open(os.path.join(base_dir, 'VERSION')).read().strip()
+    from eva_sub_cli.validators import docker_validator
+    return ' -- '.join([
+        f'eva-sub-cli: {version}',
+        f'docker image: {docker_validator.container_image}:{docker_validator.container_tag}'
```

Comment:
I'm not convince we should report the docker version image version. It might be more confusing than helpful.